runtime/queries/c-sharp/context.scm

```diff
@@ -0,0 +1,64 @@
+; Credits to nvim-treesitter-context
+
+(interface_declaration
+  body: (_) @context.end
```

Comment:
`@context.end` is not recognized. 

See [this guide for adding sticky context queries](https://github.com/SoraTenshi/helix/blob/sticky-context/book/src/guides/context.md)